docker-compose.yml

```diff
@@ -1,27 +1,28 @@
 services:
   database:
     image: postgres:17
-    restart: always #unless-stopped # change to always when in production
+    restart: always
     shm_size: 128mb
-    ports:
-      - "5432:5432"
     env_file: ".env"
 
   django:
     build: ./evergreen
-    ports:
-      - '8080:8080'
+    restart: always
     env_file: ".env"
     volumes:
     - ./media:/root/media
     depends_on:
-      - database 
+      - database
 
   nginx:
     build: ./nginx
+    restart: always
     ports:
       - '80:80'
       - '443:443'
+      - '8080:8080'
     volumes:
       - ./evergreen/staticfiles:/static
       - ./media:/media
+    depends_on:
+      - django
```

Comment:
Don't we want this the other way around? The proxy server should be up first and always filtering traffic from the app, there should never be a time when the app is running and accessible without going through the proxy first. (I am probably not fully understanding.)

Reply:
nginx can't start if django isn't running yet (it checks because of the `proxy_pass django` line, and will fail if that can't complete)